src/viperleed/gui/measure/classes/settings.py

```diff
@@ -94,7 +111,200 @@ class TooManyDefaultSettingsError(DefaultSettingsError):
+                try:
+                    value = super().get(sec, opt, raw=raw, vars=vars)
+                except (NoSectionError, NoOptionError):
+                    continue
+                else:
+                    self[section][option] = value
+                    self.remove_option(sec, opt)
+                    break
+            else:
+                value = fallback
+            if value == _UNSET:
+                raise err
+
+        if value == '':
+            try:
+                value = self._fallback[section][option]
+            except KeyError:
+                pass
+            else:
+                self[section][option] = value
+
+        return value
+
+    def prepare_aliases(self, cls_name):
+        """Prepare aliases for the handled settings.
+
+        Parameters
+        ----------
+        cls_name : str
+            Name of the class to which the loaded settings belong.
+
+        Returns
+        -------
+        None.
+        """
+        self._aliases = {}
+        self._fallback = defaultdict(dict)
+        self._cls_name = cls_name
+        if not self._cls_name:
+            # No class name, therefore we cannot look up aliases.
+            return
+        self._prepare()
+
+    def _add_new_sections(self, new_sections):
+        """Add new settings sections.
+
+        Parameters
+        ----------
+        new_sections : list
+            A list of sections to add.
+
+        Returns
+        -------
+        None.
+        """
+        for section in new_sections:
+            if section in self.sections():
+                continue
+            self.add_section(section)
+
+    def _get_aliases(self, section, option):
+        """Get aliases for a section/option pair.
+
+        Parameters
+        ----------
+        section : str
+            The section key.
+        option : str
+            The option key.
+
+        Returns
+        -------
+        aliases : list
+            A list of section/option pairs.
+        """
+        try:
+            aliases = self._aliases[section + '/' + option]
+        except KeyError:
+            return []
+        return [alias.split('/') for alias in aliases]
+
+    def _prepare(self):
+        """Prepare aliases for the handled settings.
+
+        Load relevant aliases and generate missing settings sections for
+        aliases.
+
+        Returns
+        -------
+        None.
+        """
+        conv = ast.literal_eval
+        aliases = ConfigParser()
+        aliases.read(get_aliases_path())
+
+        try:
+            relevant = conv(aliases.get(self._cls_name, 'relevant_sections'))
+        except NoSectionError:
+            # The settings for this class do not have any aliases.
+            return
+        except NoOptionError:
+            # No other relevant sections.
+            relevant = []
+
+        for section in (self._cls_name, *relevant):
+            for key in aliases[section].keys():
```

Comment:
```suggestion
            for key in aliases[section]:
```

Probably enough to iterate without `.keys()`?